hal-x86_64/src/serial.rs

```diff
@@ -122,6 +122,10 @@ impl Port {
             _is_blocking: PhantomData,
         }
     }
+
+    pub unsafe fn force_unlock(&self) {
```

Comment:
I think this change is from the other patch, can we rebase on the new master?

Reply:
whoops, yeah, i forgot to rebase, thanks for catching